metaflow/main_cli.py

```diff
@@ -324,6 +332,51 @@ def show(profile):
     else:
         echo('Configuration is set to run locally.')
 
+@configure.command(help='Exports the configuration to a file.')
+@click.option('--profile', '-p', default='',
+              help="Optional user profile whose configuration must be "
+                   "exported.")
+@click.argument('output_filename', type=click.Path(resolve_path=True))
+def export(profile, output_filename):
```

Comment:
This fails when no profile is set.
```
➜  metaflow git:(crk) metaflow configure export tmp
Metaflow (2.0.0)

Traceback (most recent call last):
  File "/usr/local/bin/metaflow", line 11, in <module>
    load_entry_point('metaflow', 'console_scripts', 'metaflow')()
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Users/savin/code/metaflow/metaflow/main_cli.py", line 345, in export
    with open(path, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/savin/.metaflowconfig/config.json'
```

Reply:
Thanks for reporting this. I fixed this in the latest revision, by exporting the empty_dict for this case.